ammonia_test_script.py

```diff
@@ -0,0 +1,40 @@
+#####################################################################
+# This script tests the ammonia fitter                              #
+# with fake data produced by ammonia_model.n_modelfunc              #
+# using the xarr from G031.947+00.076_nh3_11_Tastar.fits [for now]  #
+#####################################################################
+
+from pyspeckit.spectrum.models import ammonia
+from pyspeckit import Spectrum
+from pyspeckit.spectrum.units import SpectroscopicAxis
+from astropy import units as u
+import numpy as np
+import matplotlib.pyplot as plt
+
+parameter_error_amplitude = [5., 5., 5., 5., 5., 5.]
+params = [2.3,2,0.,0.84,0.2,0.43]
+
+sp = Spectrum('G031.947+00.076_nh3_11_Tastar.fits')
+xarr = SpectroscopicAxis(np.linspace(-250, 385, 8192), refX=23694500000.0*u.Hz)
```

Comment:
It is given the default value unit='Hz' in the SpectroscopicAxis' `__new`__ method. Also, as it is written now the user must specify that unit=None or 'unknown' in order for xarr.unit to be set to u.dimensionless_unscaled, which isn't very intuitive. Should I change this?